README.md

```diff
@@ -18,26 +22,31 @@ const MY_SERVICE_META = {
   multiArg: false, // defaults to false. If true accepts an array for arguments, if false an array will be assumed to be the first (and only) argument.
   expose: [
     // The methods to be exposed publicly
-    // BREAKING CHANGE: expose now requires an object
-    { methodName: "doStuff" },
+
     {
       methodName: "moreStuff",
       methodTimeout: 15000,
       paramNames: ["stuffs"],
-      help: "This is a silly method"
-    }
-  ]
+      help: "This is a silly method",
+    },
+  ],
 };
 
-const myRpcService = lokeHttpRpc.createRequestHandler(
-  myService,
-  MY_SERVICE_META
-);
+//BREAKING CHANGES:
+// "createRequestHandler" was renamed to createWellKnownHandler and can be accesed through register()
```

Comment:
No, this is wrong. It is still createRequestHandler but you need to register the service first.

Reply:
Fixed this